packages/venia-concept/src/reducers/cart.js

```diff
@@ -35,6 +35,18 @@ const reducerMap = {
             ...payload
         };
     },
+    [actions.removeItem.receive]: (state, { payload, error }) => {
+        if (error) {
+            return initialState;
+        }
+        if (payload.cartItemCount == 1) {
+            return initialState;
+        }
```

Comment:
In the `removeItem` action creator, you left a comment about how it was necessary to clear the cart id when removing the last item. Is this the same situation here? If so, consider leaving a similar comment.